scripts/acn/k8s_deploy_acn_node.py

```diff
@@ -0,0 +1,638 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+""" Deploy an ACN libp2p node to a kubernetes cluster """
+
+import argparse
+import base64
+import os
+import subprocess  # nosec
+import sys
+from datetime import datetime
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Tuple, Type
+
+SCRIPT_DIR = os.path.abspath(os.path.dirname(os.path.realpath(__file__)))
+
+sys.path.append(SCRIPT_DIR)
+from run_acn_node_standalone import (  # noqa # pylint: disable=wrong-import-position
```

Comment:
reuse `run_acn_node_standalone.py` script checks.
This allows to save time by detecting node configuration errors before pushing the image and running the container on k8s cluster